### PR TITLE
Add .rubocop.yml and apply fixes

### PR DESCRIPTION
**Pull request by iamthemuffinman**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -1,7 +1,6 @@
-require "bundler/gem_tasks"
-require "rspec/core/rake_task"
+require 'bundler/gem_tasks'
+require 'rspec/core/rake_task'
 
 RSpec::Core::RakeTask.new(:spec)
 
-task :default => :spec
-
+task default: :spec
```
Comment (raskchanky) [inline Rakefile]: 😞 
❤️ #️⃣ 🚀

Comment (iamthemuffinman) [inline Rakefile]: What does a sad face, heart, hashtag, and a rocket ship mean? If I had to take a guess, I would say it means revert the change you pasted above and then it looks good?

Comment (raskchanky) [inline Rakefile]: It just means I like the hash rocket syntax over the other.  😁   I think it's more readable.

Comment (iamthemuffinman) [inline Rakefile]: Ahh. I gotcha. Brb. :)

**Comment by iamthemuffinman** [comment]
@raskchanky That look better? 😄

**Comment by iamthemuffinman** [comment]
I had the wrong name and email in there. Wanted to take care of it before merge.

**Comment by raskchanky** [comment]
@iamthemuffinman Looks better to me 😁
